package exception;

public class Exception01 {
	public static void main(String[] args) {
		System.out.println("100 ÷ 0 は？");
		int resoult = 100 / 0;
		System.out.println("計算結果" + resoult);
		System.out.println("プログラム終了");
	}
}
